### DISCUSSION TITLE
Welcome to Hactoberfest-X-MIBCS-2022 Discussions!

### DISCUSSION THREAD

**Discussion by Chinmay-03** [Chinmay-03/Hacktoberfest-X-MIBCS-2022#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Saiesh-Agre** [comment]
You can search for repository on your favorite topics and post them on the tasks. You can then open the repo by yourself to explore it
